enterprise/admin/forms.py

```diff
@@ -566,6 +566,18 @@ def clean(self):
         """
         cleaned_data = super(EnterpriseCustomerReportingConfigAdminForm, self).clean()
         report_customer = cleaned_data.get('enterprise_customer')
+        data_type = cleaned_data.get('data_type')
+
+        if data_type in EnterpriseCustomerReportingConfiguration.PEARSON_ONLY_REPORTS \
```

Comment:
please add a comment explaining that this code will be removed after the partial rollout.